src/shared/lib/encryption/EncryptionManager.test.ts

```diff
@@ -135,7 +138,9 @@ describe('Encryption', () => {
 
       const generatedPrivateKey = encryptionManager.getPrivateKey(userParams);
 
-      expect(generatedPrivateKey).toStrictEqual(result);
+      const expectedPrivateKey = encryptionManager.getPrivateKey(userParams);
```

Comment:
The way you updated this test is incorrect now.
`expectedPrivateKey` and `generatedPrivateKey` are both the result from the same call to `encryptionManager.getPrivateKey`